nrp-proxy tests fail (not starting)

The `npm test` fails with error

```
> JUNIT_REPORT_PATH=junit.xml nyc mocha test/*/*

/workspaces/nrp-proxy/node_modules/nock/node_modules/chai/lib/chai/assertion.js:69
    flag(this, 'eql', config.deepEqual ?? util.eql);
                                        ^

SyntaxError: Unexpected token ?
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Module.replacementCompile (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at Module._extensions..js (module.js:664:10)
    at /workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/workspaces/nrp-proxy/node_modules/ts-node/src/index.ts:431:14)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nock/node_modules/chai/lib/chai.js:63:17)
    at Module._compile (module.js:653:30)
    at Module.replacementCompile (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at Module._extensions..js (module.js:664:10)
    at /workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/workspaces/nrp-proxy/node_modules/ts-node/src/index.ts:431:14)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nock/node_modules/chai/index.js:1:18)
    at Module._compile (module.js:653:30)
    at Module.replacementCompile (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at Module._extensions..js (module.js:664:10)
    at /workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/workspaces/nrp-proxy/node_modules/ts-node/src/index.ts:431:14)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nock/lib/back.js:11:14)
    at Module._compile (module.js:653:30)
    at Module.replacementCompile (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at Module._extensions..js (module.js:664:10)
    at /workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/workspaces/nrp-proxy/node_modules/ts-node/src/index.ts:431:14)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nock/index.js:12:23)
    at Module._compile (module.js:653:30)
    at Module.replacementCompile (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at Module._extensions..js (module.js:664:10)
    at /workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/workspaces/nrp-proxy/node_modules/ts-node/src/index.ts:431:14)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/workspaces/nrp-proxy/test/proxy/ExperimentServiceFactory.js:7:12)
    at Module._compile (module.js:653:30)
    at Module.replacementCompile (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at Module.m._compile (/workspaces/nrp-proxy/node_modules/ts-node/src/index.ts:439:23)
    at Module._extensions..js (module.js:664:10)
    at /workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4
    at require.extensions.(anonymous function) (/workspaces/nrp-proxy/node_modules/ts-node/src/index.ts:442:12)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at /workspaces/nrp-proxy/node_modules/mocha/lib/mocha.js:334:36
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (/workspaces/nrp-proxy/node_modules/mocha/lib/mocha.js:331:14)
    at Mocha.run (/workspaces/nrp-proxy/node_modules/mocha/lib/mocha.js:809:10)
    at Object.exports.singleRun (/workspaces/nrp-proxy/node_modules/mocha/lib/cli/run-helpers.js:108:16)
    at exports.runMocha (/workspaces/nrp-proxy/node_modules/mocha/lib/cli/run-helpers.js:142:13)
    at Object.exports.handler.argv [as handler] (/workspaces/nrp-proxy/node_modules/mocha/lib/cli/run.js:292:3)
    at Object.runCommand (/workspaces/nrp-proxy/node_modules/yargs/lib/command.js:242:26)
    at Object.parseArgs [as _parseArgs] (/workspaces/nrp-proxy/node_modules/yargs/yargs.js:1096:28)
    at Object.parse (/workspaces/nrp-proxy/node_modules/yargs/yargs.js:575:25)
    at Object.exports.main (/workspaces/nrp-proxy/node_modules/mocha/lib/cli/cli.js:68:6)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/mocha/bin/mocha:162:29)
    at Module._compile (module.js:653:30)
    at Module.replacementCompile (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:58:13)
    at Module._extensions..js (module.js:664:10)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/node_modules/append-transform/index.js:62:4)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at runMain (/root/.node-spawn-wrap-19120-e59c790ffbf9/node:68:10)
    at Function.<anonymous> (/root/.node-spawn-wrap-19120-e59c790ffbf9/node:171:5)
    at Object.<anonymous> (/workspaces/nrp-proxy/node_modules/nyc/bin/wrap.js:23:4)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at /root/.node-spawn-wrap-19120-e59c790ffbf9/node:178:8
    at Object.<anonymous> (/root/.node-spawn-wrap-19120-e59c790ffbf9/node:181:3)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
npm ERR! Test failed.  See above for more details.
```

--- a/package.json
+++ b/package.json
@@ -50,7 +50,7 @@
     "lint-staged": "4.3.0",
     "mocha": "^6.2.3",
     "mocha-jenkins-reporter": "^0.4.8",
-    "nock": "^9.1.6",
+    "nock": "9.1.6",
     "prettier": "^1.16.4",
     "sinon": "^2.4.1",
     "ts-node": "^7.0.1",

